### PR TITLE
Update resource debug page (jsp) 

### PR DESCRIPTION
**Pull request by jadelkhoury**
Update resource debug page (jsp) to
(1) link to its shape
(2) link to its rdf:type(s)
(3) list its extendedProperties - properties that are not necessarily
defined in the getters/setters of the Java classes.

### CODE REVIEW

File: org.eclipse.lyo.oslc4j.codegenerator/src/org/eclipse/lyo/oslc4j/codegenerator/jsp/generateGetResourceJsp.mtl [hunk 1]
```diff
@@ -107,12 +116,26 @@
...
+        URI shapeUri = UriBuilder.fromUri(OSLC4JUtils.getServletURI()).path(OslcConstants.PATH_RESOURCE_SHAPES).path([javaInterfaceNameForConstants(aResource.definingDomainSpecification())/].[resourcePathConstantName(aResource)/]).build();
+        out.write("<a href=\"" + shapeUri + "\">" + shapeUri + "</a>");
+        %>
+        </p>
+        <p class="lead">rdf:type(s):</p>
+        <ul>
+        <%
+        Collection<URI> types = a[javaName(aResource, true)/].getTypes();   
+        for (URI type : a[javaName(aResource, true)/].getTypes()) {
+            out.write("<li><a href=\"" + type + "\">" + type + "</a></li>");
```
Comment (berezovskyi) [inline org.eclipse.lyo.oslc4j.codegenerator/src/org/eclipse/lyo/oslc4j/codegenerator/jsp/generateGetResourceJsp.mtl]: Same here

File: org.eclipse.lyo.oslc4j.codegenerator/src/org/eclipse/lyo/oslc4j/codegenerator/jsp/generateGetResourceJsp.mtl [hunk 2]
```diff
@@ -107,12 +116,26 @@
...
     <div class="page-header">
         <h1>[aResource.name /] resource</h1>
-        <p class="lead">URI:&nbsp;
+        <p class="lead">rdf:about:&nbsp;
         <jsp:include page="[resourceToHtmlJspRelativeFileName(aResource, contextAdaptorInterface)/]"></jsp:include>
         </p>
+        <p class="lead">shape:&nbsp;
+        <%
+        URI shapeUri = UriBuilder.fromUri(OSLC4JUtils.getServletURI()).path(OslcConstants.PATH_RESOURCE_SHAPES).path([javaInterfaceNameForConstants(aResource.definingDomainSpecification())/].[resourcePathConstantName(aResource)/]).build();
+        out.write("<a href=\"" + shapeUri + "\">" + shapeUri + "</a>");
```
[the remark this answers is not in the document]
Comment (jadelkhoury) [reply org.eclipse.lyo.oslc4j.codegenerator/src/org/eclipse/lyo/oslc4j/codegenerator/jsp/generateGetResourceJsp.mtl]: What's the "ouch" for? a more constructive comment please? :-)

File: org.eclipse.lyo.oslc4j.codegenerator/src/org/eclipse/lyo/oslc4j/codegenerator/jsp/generateGetResourceJsp.mtl (hunk 1 above)
Comment (jadelkhoury) [reply org.eclipse.lyo.oslc4j.codegenerator/src/org/eclipse/lyo/oslc4j/codegenerator/jsp/generateGetResourceJsp.mtl]: same as in "ouch" or "awesome"?

**Comment by jadelkhoury** [comment]
How about now @berezovskyi . Good enough?

**Pull request merged**